apps/registration-api/RegistrationApi.js

```diff
@@ -18,6 +18,7 @@ app.use(morgan('combined'))
 
 app.get('/register', regAuthCheck, (req, res) => {
 
+  const userName = req.get('User')
```

Comment:
The problem with this solution is it can be modified by the user.  Please the `req.username` that was already added to the middleware.